src/components/play-pause/play-pause.tsx

```diff
@@ -92,7 +93,8 @@ class PlayPause extends Component<any, any> {
   render(props: any): VNode<any> | undefined {
     const controlButtonClass = this.props.isPlayingAdOrPlayback ? [style.controlButton, style.isPlaying].join(' ') : style.controlButton;
     const isStartOver = props.isPlaybackEnded && !this.props.adBreak;
-    const playbackStateText = this.props.isPlayingAdOrPlayback ? this.props.pauseText : this.props.playText;
+    const entryName = `, ${this.props.title}: ${this.props.player.sources.metadata?.name}`;
+    const playbackStateText = `${this.props.isPlayingAdOrPlayback ? this.props.pauseText : this.props.playText}${entryName}`;
```

Comment:
no space between the strings ?

Reply:
there is, when you use `` so space between the variables adding also space in the string